docs/experience/configure-requirements.md

```diff
@@ -0,0 +1,34 @@
+title: Configure Requirements
+description: Guide on how to configure Sematext Experience
```

Comment:
description: Guide on how to configure Sematext Experience
->
description: A guide on how to configure Sematext Experience with custom time thresholds for page loads and HTTP requests. Easily configure On-Page Transactions and URL Groups as well.